firefox_ui_harness/default_prefs.py

```diff
@@ -36,6 +36,7 @@
     'extensions.showMismatchUI': False,
     'extensions.update.enabled': False,
     'extensions.update.notifyUser': False,
+    'focusmanager.testmode': True,
```

Comment:
Please check which other prefs we might miss compared to mozmill, which should be set by Marionette. We might also want to add them here for safety.

Regarding the above preference I would ask David if we even not have to enable it for Firefox Desktop by default.